spec/promisifySpec.js

```diff
@@ -63,6 +63,22 @@ describe('promisify', function() {
     expect(await promisify(ex)()).toBe('surprise!');
   });
 
+  describe('method', function() {
+    it('should promisify the given method', async function() {
+      const api = {
+        beep(done) {
```

Comment:
Can you make the implementation of `beep` use `this`, to make sure that the binding works?